<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>明日方舟抽卡计算器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_input">
      <property name="title">
       <string>输入当前资源</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_ten_ticket">
         <property name="text">
          <string>十连寻访凭证：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox_ten_ticket">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_normal_ticket">
         <property name="text">
          <string>普通寻访凭证：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_normal_ticket">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_originium">
         <property name="text">
          <string>源石数量：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_originium">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_orundum">
         <property name="text">
          <string>合成玉数量：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_orundum">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_calculation">
      <property name="title">
       <string>计算结果</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_total_pulls">
         <property name="text">
          <string>总抽卡次数：0</string>
         </property>
         <property name="styleSheet">
          <string>font-size: 14px; font-weight: bold;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ten_pulls">
         <property name="text">
          <string>十连抽次数：0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_single_pulls">
         <property name="text">
          <string>单抽次数：0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_remaining_orundum">
         <property name="text">
          <string>剩余合成玉：0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_remaining_originium">
         <property name="text">
          <string>剩余源石：0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_info">
      <property name="title">
       <string>计算说明</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_info">
         <property name="text">
          <string>• 1个十连寻访凭证 = 10次抽卡
• 1个普通寻访凭证 = 1次抽卡
• 600合成玉 = 1次抽卡
• 1源石 = 180合成玉
• 优先使用十连凭证，其次普通凭证，最后使用合成玉</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
